console/src/components/Services/Events/Common/Components/EventsTable.tsx

```diff
@@ -25,10 +29,14 @@ const CancelEventButton: React.FC<CancelButtonProps> = ({
 }) => (
   <Button
     key={id}
+    className={styles.add_mar_right_small}
     onClick={onClickHandler}
     color="white"
     size="xs"
     title="Cancel Event"
+    onMouseDown={(e: React.MouseEvent) => {
```

Comment:
No need for typings here. This will be inferred automatically. 
```suggestion
    onMouseDown={(e) => {
```